### PR TITLE
feat: create onboarding section for the sponsor dashboard

### PR DESCRIPTION
**Pull request by ashaduzzamane**
### Tickets:

- HCK-

### List of changes:

- Create the onboarding section for the sponsor dashboard
- Comprised of three tabs: general onboarding, discord onboarding, and schedule

### Type of change:

Please delete options that aren't relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### PR Checklist:

- [x] Merged `develop` branch (before testing)
- [ ] Linted my code locally
- [ ] Listed change(s) in the Changelog
- [ ] Tested all links in project relevant browsers
- [ ] Tested all links on different screen sizes
- [ ] Referenced all useful info (issues, tasks, etc)

### Screenshots:
Web:
image

image

Mobile: 
image

image

### CODE REVIEW

**Comment by pierreTklein** [comment]
Can you post web + mobile screenshots of the new page?

**Comment by ashaduzzamane** [comment]
> Can you post web + mobile screenshots of the new page?

done! sorry, I forgot to include that. I also made a few minor changes

**Comment by logan-r** [comment]
image

Proper navbar don't display and clicking link redirects me to the home page? Maybe has something to do with email verification

**Review by logan-r — APPROVED** [review]
Looks great!

File: src/features/Onboarding/DiscordOnboarding.tsx [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+                        <li>
+                            <b>Pro Tip: </b>You can rearrange your server list by dragging server icons up and down
+                        </li>
+                    </ul>
+                    <h2>How do I make an account?</h2>
+                    <p>If you're on browser:</p>
+                    <ol>
+                        <li>
+                            Visit{" "} 
+                            <a
```
Comment (pierreTklein) [inline src/features/Onboarding/DiscordOnboarding.tsx:60]: Consider LinkDuo component

File: src/features/Onboarding/DiscordOnboarding.tsx [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+                        <li>
+                            <p>Change notifications from all to mentions only: </p>
+                            <StyledGIF src={NotifsToMentionGIF} />
+                            <br></br>
+                        </li>
+                    </ul>
+                    <h2>Helpful Online Resources</h2>
+                    <ul>
+                        <li>
+                            <a
```
Comment (pierreTklein) [inline src/features/Onboarding/DiscordOnboarding.tsx:227]: Ditto

File: src/features/Onboarding/DiscordOnboarding.tsx [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+                                href="https://support.discord.com/hc/en-us/sections/360008206871-Discord-Basics"
+                                target="_blank"
+                                rel="noopener noreferrer"
+                            >
+                                https://support.discord.com/hc/en-us/sections/360008206871-Discord-Basics
+                            </a>
+                        </li>
+                        <li>
+                            <a
```
Comment (pierreTklein) [inline src/features/Onboarding/DiscordOnboarding.tsx:236]: Ditto (last ditto, check rest of file)

File: src/features/Onboarding/GeneralOnboarding.tsx [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+                    communicate with you on.
+                    <br></br>
+                    <br></br>
+                    On the day of, sponsor check-in will start at <b>6:00 p.m. </b>on Discord. Please 
+                    check in with your coordinator when you arrive.
+                    <br></br>
+                    <br></br>
+                    If part of your package involves speaking at the opening ceremonies, please ensure 
+                    your video is uploaded to{" "}
+                    <a
```
Comment (pierreTklein) [inline src/features/Onboarding/GeneralOnboarding.tsx:35]: Ditto

File: src/features/Onboarding/DiscordOnboarding.tsx (hunk 1 above)
Comment (ashaduzzamane) [reply src/features/Onboarding/DiscordOnboarding.tsx]: done

**Review by logan-r — APPROVED** [review]

**Pull request merged**